tests/requirements.txt

```diff
@@ -0,0 +1,7 @@
+pytest
```

Comment:
We have already moved this file to https://github.com/dell/ansible-powerstore/blob/main/tests/unit/requirements.txt 

I feel we can remove this tests/requirements.txt